### PR TITLE
changed wa_fields to CAM fields

### PR DESCRIPTION
**Pull request by nickhill111**
SSCS-10838

**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: benefit/data/sheets/CaseTypeTab/CaseTypeTab-nonprod.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 [
   {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "summary_sscs", "TabLabel": "Summary", "TabDisplayOrder": 1, "CaseFieldID": "pcqId", "TabFieldDisplayOrder": 31},
-  {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "wa_fields", "TabLabel": "Appeal Details", "TabDisplayOrder": 29, "CaseFieldID": "SearchCriteria", "TabFieldDisplayOrder": 7}
+  {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "cam_fields", "TabLabel": "Appeal Details", "TabDisplayOrder": 29, "CaseFieldID": "SearchCriteria", "TabFieldDisplayOrder": 7}
```
Comment (dattipoeHMCTS) [inline benefit/data/sheets/CaseTypeTab/CaseTypeTab-nonprod.json:3]: Does the label need to change?

Comment (TeaWithLucas) [reply benefit/data/sheets/CaseTypeTab/CaseTypeTab-nonprod.json:3]: Yeah the label needs to change to CAM fields

File: benefit/data/sheets/CaseTypeTab/CaseTypeTab.json [hunk 2]
```diff
@@ -268,10 +268,10 @@
...
-  {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "wa_fields", "TabLabel": "Appeal Details", "TabDisplayOrder": 29, "CaseFieldID": "caseNameHmctsRestricted", "TabFieldDisplayOrder": 2},
-  {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "wa_fields", "TabLabel": "Appeal Details", "TabDisplayOrder": 29, "CaseFieldID": "caseNamePublic", "TabFieldDisplayOrder": 3},
-  {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "wa_fields", "TabLabel": "Appeal Details", "TabDisplayOrder": 29, "CaseFieldID": "ogdType", "TabFieldDisplayOrder": 4},
-  {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "wa_fields", "TabLabel": "Appeal Details", "TabDisplayOrder": 29, "CaseFieldID": "caseManagementCategory", "TabFieldDisplayOrder": 5},
-  {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "wa_fields", "TabLabel": "Appeal Details", "TabDisplayOrder": 29, "CaseFieldID": "CaseAccessCategory", "TabFieldDisplayOrder": 6}
+  {"LiveFrom": "", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-superuser", "TabID": "cam_fields", "TabLabel": "CAM fields", "TabDisplayOrder": 29, "CaseFieldID": "caseNameHmctsInternal", "TabFieldDisplayOrder": 1},
```
Comment (TeaWithLucas) [inline benefit/data/sheets/CaseTypeTab/CaseTypeTab.json:271]: The label from these needs to change to CAM fields

**Review by dattipoeHMCTS — DISMISSED** [review]

File: benefit/data/sheets/CaseTypeTab/CaseTypeTab-nonprod.json [hunk 3]
```diff
@@ -1,4 +1,4 @@
 [
   {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "summary_sscs", "TabLabel": "Summary", "TabDisplayOrder": 1, "CaseFieldID": "pcqId", "TabFieldDisplayOrder": 31},
-  {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "wa_fields", "TabLabel": "Appeal Details", "TabDisplayOrder": 29, "CaseFieldID": "SearchCriteria", "TabFieldDisplayOrder": 7}
+  {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "cam_fields", "TabLabel": "CAM fields", "TabDisplayOrder": 29, "CaseFieldID": "SearchCriteria", "TabFieldDisplayOrder": 7}
```
Comment (TeaWithLucas) [inline benefit/data/sheets/CaseTypeTab/CaseTypeTab-nonprod.json:3]: Minor: Capitalise Fields in "CAM fields"

**Review by TeaWithLucas — APPROVED** [review]

**Review by bensonlievoco — APPROVED** [review]

**Review by dattipoeHMCTS — APPROVED** [review]

**Pull request merged**